Elm.gitignore

```diff
@@ -0,0 +1,2 @@
+elm-stuff/*
+!elm-stuff/exact-dependencies.json
```

Comment:
elm-stuff/ definitely is all build artifacts. I'm not certain about the exact-dependencies.json file, but I think anyone doing production quality Elm would want to keep track of that.